src/index.tsx

```diff
@@ -3,6 +3,7 @@ import ReactDOM from 'react-dom/client';
 import './index.scss';
 import { BrowserRouter } from 'react-router-dom';
 import { QueryClient, QueryClientProvider } from 'react-query';
+import { Provider } from 'jotai';
```

Comment:
사소하지만 명시적으로 JotaiProvider 임을 드러냈으면 좋겠어요
```suggestion
import { Provider as JotaiProvider } from 'jotai';
```